Adding VMC provider kb and adding Registering Instances to Your vCenter/VMS Provider sections (#2956)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/Using-CenturyLink-Private-Cloud-with-VMC-on-AWS.md

{{{ "title": "Using the CenturyLink Private Cloud with VMC on AWS",
"date": "12-05-2019",
"author": "Sharon Wang",
"keywords": ["cam", "cloud application manager", "vmware", "aws", "vmc", "cpc", "centurylink private cloud"],
"attachments": [],
"contentIsHTML": false
}}}

**In this article:**
* [Overview](#overview)
* [Audience](#audience)
* [Prerequisites](#prerequisites)
* [Required Permissions](#required-permissions)
* [Bootstrapping virtual machine templates with the Cloud Application Manager Agent](#bootstrapping-virtual-machine-templates-with-the-cloud-application-manager-agent)
* [Registering a VMC provider in Cloud Application Manager](#registering-a-vmc-provider-in-cloud-application-manager)
* [Registering Instances to Your VMC Provider](#registering-instances-to-your-vmc-provider)
* [Deploying in Your VMC](#deploying-in-your-vmc)
* [Getting General Support](#getting-general-support)

### Overview

This article is meant to assist users of Cloud Application Manager to configure a CenturyLink Private Cloud with VMC on AWS provider, then deploy or register a virtual machines with it.

### Audience

> All users of Cloud Application Manager using CenturyLink Private Cloud with VMC on AWS providers.

Reviewer: Please, use the official name `CenturyLink Private Cloud for VMware Cloud on AWS`
```suggestion
All users of Cloud Application Manager using CenturyLink Private Cloud for VMware Cloud on AWS (CPC for VMC) providers.
```

### Prerequisites

* An active *Cloud Application Manager* account

> * Access and credentials to a **CenturyLink Private Cloud with VMC on AWS**.

Reviewer: ```suggestion
* Access and credentials to a **CenturyLink Private Cloud for VMware Cloud on AWS** environment.
```

   * **Note:** If running Cloud Application Manager as an appliance, replace cam.ctl.io with the appliance hostname or IP address.

4. Right-click the script and click **Run PowerShell**.

### Registering a CenturyLink Private Cloud with VMC on AWS provider in Cloud Application Manager

In order to deploy to a VMC on AWS, you must first provide information to connect. 

**Steps**

1. In Cloud Application Manager, click **Providers** > **New Provider**.

2. Select **CenturyLink Private Cloud with VMC on AWS**.

3. Enter the endpoint URL for the VMC server and a username, password to the VMC API.
   * **Note:** The endpoint URL can be a hostname or an IP address, and has to be prefixed with **https://**. Be sure to use https and not http.

   ![Creating a new VMC provider](../../images/cloud-application-manager/VMC01.png)

### Registering Instances to Your VMC Provider

You can also register you exsiting instances to CAM portal. First click on **Unregistered Instances** Tab, here you can see all the exsting instances in you VMC that you can import into your CAM portal.

Simply select one server and click on the down arrow button on the right side.

> ![register_to_vcnter1](../../images/cloud-application-manager/deploying-anywhere/register_to_vmc1.png)

Reviewer: ```suggestion
![Register a vCenter instance](../../images/cloud-application-manager/deploying-anywhere/register_to_vmc1.png)
```

Name instance to be registered and click on **Register**

> ![register_to_vcnter1](../../images/cloud-application-manager/deploying-anywhere/register_to_vmc2.png)

Reviewer: ```suggestion
![Name instance to be registered](../../images/cloud-application-manager/deploying-anywhere/register_to_vmc2.png)
```

Connect to server and execute the script below with admin privileges.

> ![register_to_vcnter1](../../images/cloud-application-manager/deploying-anywhere/register_to_vmc3.png)

Reviewer: ```suggestion
![Script to register the instance](../../images/cloud-application-manager/deploying-anywhere/register_to_vmc3.png)
```
